ui/k-info-panel/switch_info.kytos

```diff
@@ -76,3 +102,37 @@
  }
 </script>
 
+<style>
+.metadata_table {
+  color: #ccc;
+  max-height: 250px;
+  text-align: center;
+  margin: 0 auto;
+  display: block;
+  padding: 0.5em 0 1em 0.3em;
+  font-size: 0.8em;
+  height: 190px;
```

Comment:
Need an overflow property for table with static height value, for the cases with too many data. You can test adding 20 more fake metadata_items to check how it is rendered.

Reply:
Now it handles overflow. Commit [36ffa8d66af7004f9e4d43f478c5f798ab9da12b](https://github.com/kytos-ng/topology/pull/75/commits/36ffa8d66af7004f9e4d43f478c5f798ab9da12b)